org.mwc.debrief.core/src/org/mwc/debrief/core/loaders/BRTLoader.java

```diff
@@ -155,8 +156,17 @@ public void run(final IProgressMonitor pm)
           final BRTImporter importer = new BRTImporter();
           final TrackWrapper[] allTracks = BRTImporter.getTracks(theLayers);
           final TrackWrapper theTrack = BRTImporter.findTrack(allTracks);
+          
+
+          // create sensor
+          String fileName = new File(_fileName).getName();
+          if (fileName.lastIndexOf('.') > 0)
+          {
+            fileName = fileName.substring(0, fileName.lastIndexOf('.'));
+          }
+          
           final ImportBRTDialog wizard = new ImportBRTDialog(theTrack,
```

Comment:
@saulhidalgoaular - we've got two very similar variable names here, and I suspect that on line 190 you meant to refer to `_fileName`.

I suggest that `fileName` change to `defaultSensorName`.  Then it will be clear which is to be used in #190..

Reply:
Ok. Changing now!!